ocfweb/stats/templates/stats/summary.html

```diff
@@ -11,6 +11,7 @@
             <img class="img-responsive" src="{% url 'daily_graph_image' %}" />
             <img class="img-responsive" src="{% url 'session_count_image' %}" />
             <img class="img-responsive" src="{% url 'session_length_image' %}" />
+            <img class="img-responsive" src="{% url 'daily_job_image' %}" />
```

Comment:
That's a different thing (but that's a better url anyway), I was talking about putting this image in the page at [/stats/printing/](http://supernova.ocf.berkeley.edu:8977/stats/printing/), not at the current page ([/stats/](http://supernova.ocf.berkeley.edu:8977/stats/)).